app/views/inputSingleCheckbox.scala.html

```diff
@@ -8,19 +8,23 @@
 @elements = @{new FieldElements(field.id, field, null, args.toMap, messages) }
 @value = {@elements.args.get(Symbol("_value"))}
 @label = {@elements.args.get(Symbol("_label"))}
+@isSelected = {@elements.args.selectedInput(value.toString)}
```

Comment:
@suzannehamilton this is the area which doesn't feel like its the best way to get access to checked/unchecked in when rendering the checkboxes.